# Disables a check for 64-bit offset types because it breaks Emscripten and PlayStation Vita builds.

--- a/cmake/SndFileChecks.cmake
+++ b/cmake/SndFileChecks.cmake
@@ -113,13 +113,9 @@ else ()
 		set (SF_COUNT_MAX "0x7FFFFFFFFFFFFFFFLL")
 		set (SIZEOF_SF_COUNT_T 8)
 	else ()
-		message ("")
-		message ("*** The configure process has determined that this system is capable")
-		message ("*** of Large File Support but has not been able to find a type which")
-		message ("*** is an unambiguous 64 bit file offset.")
-		message ("*** Please contact the author to help resolve this problem.")
-		message ("")
-		message (FATAL_ERROR "Bad file offset type.")
+		set (TYPEOF_SF_COUNT_T "int64_t")
+		set (SF_COUNT_MAX "0x7FFFFFFFFFFFFFFFLL")
+		set (SIZEOF_SF_COUNT_T 8)
 	endif ()
 endif ()
 
